calebwagner/python-server
# created request.py packages for animals, employees, customers lists

#### Changes Made
1. Added file `request.py` to `employees` directory.
1. Modified file `employees/request.py` to include `employees` package.
1. Added file `request.py` to `customers` directory.
1. Modified file `customers/request.py` to include `customers` package.
1. Added file `request.py` to `aniamls` directory.
1. Modified file `animals/request.py` to include `animals` package.
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout cjw-ch3
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > play python server
    > check packages for added changes
    > in Postman check for route parameters:
- localhost:8088/animals
- localhost:8088/locations
- localhost:8088/animals

4. View code file.
    > Confirm file modifications are present as indicated above.
